processes/local_development/seed_test_data.py

```diff
@@ -47,7 +47,15 @@ def runProcess(self):
             self.db_manager.close_connection()
 
     def save_test_record(self):
-        test_record = Record(**self.test_data)
-
-        self.db_manager.session.add(test_record)
-        self.db_manager.session.commit()
+        existing_record = self.db_manager.session.query(Record).filter_by(uuid=self.test_data['uuid']).first()
+
+        if existing_record:
+            for key, value in self.test_data.items():
+                if hasattr(existing_record, key):
```

Comment:
I also recommend not setting the uuid differently every time.

Reply:
Like you want us to give a value to UUID manually during creation?

Reply:
Meaning we should not update the uuid here. it should remain the same!